src/options.js

```diff
@@ -11,8 +13,16 @@ var defaults = {
   bracketSpacing: true
 };
 
+var exampleConfig = Object.assign({}, defaults, {
+  filename: "testFilename",
+  useFlowParser: false,
+  printWidth: 80,
+  originalText: "text"
+});
+
 // Copy options and fill in default values.
 exports.normalize = function(options) {
+  validate(options, { exampleConfig });
```

Comment:
Generally I think that assigning `originalText` and all non-defaults should be responsibility of `normalize`